src/snowflake/cli/_plugins/nativeapp/entities/application_package_child_interface.py

```diff
@@ -0,0 +1,43 @@
+from abc import ABC, abstractmethod
+from pathlib import Path
+from typing import Optional
+
+
+class ApplicationPackageChildInterface(ABC):
+    @abstractmethod
+    def bundle(self, bundle_root=Path, *args, **kwargs) -> None:
```

Comment:
Isn't it an action? So the name should be action_bundle?

Reply:
I intentionally separated it from the action to be able to add additional logic only to the action. We can make it the actual action if we'll see it's unnecessary.